src/Todoist.Net/Models/BaseUnsetEntity.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text.Json.Serialization;
+
+namespace Todoist.Net.Models
+{
+    /// <summary>
+    /// Represents a base entity that implements <see cref="IUnsettableProperties"/>.
+    /// </summary>
+    /// <seealso cref="IUnsettableProperties"/>
+    public class BaseUnsetEntity : BaseEntity, IUnsettableProperties
```

Comment:
Probably this class should be an abstract one?

Reply:
Yes, I missed that.